src/course-page-resources/course-page/CoursePageConfigCard.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@edx/paragon';
+import classNames from 'classnames';
+
+import messages from '../messages';
+
+function CoursePageConfigCard({
+  intl,
+  id,
+  title,
+  description,
+  pageStatus,
+  showSettings,
+}) {
+  const cardSettings = 'bg-white shadow p-3 m-0 my-3 course-page-title';
+  const cardClasses = classNames('card', cardSettings, { disabled: !pageStatus });
+  const cardSubSettings = 'bg-white px-3 py-0';
```

Comment:
They probably should be styles rather than settings. I was stuck for naming at the time.

Also, I am no longer using the card class, but still referring to the objects as cards. However, I am still in the process of cleaning up that area of the code.

Reply:
Actually, this has been cleaned up entirely.